src/packet_analysis/Analyzer.cc

```diff
@@ -168,26 +168,30 @@ void Analyzer::Weird(const char* name, Packet* packet, const char* addl) const
 
 void Analyzer::AnalyzerConfirmation(session::Session* session, zeek::Tag arg_tag)
 	{
-	if ( session->AnalyzerState(arg_tag) == session::AnalyzerConfirmationState::CONFIRMED )
+	auto const& effective_tag = arg_tag ? arg_tag : GetAnalyzerTag();
+
+	if ( session->AnalyzerState(effective_tag) == session::AnalyzerConfirmationState::CONFIRMED )
 		return;
 
-	session->SetAnalyzerState(GetAnalyzerTag(), session::AnalyzerConfirmationState::CONFIRMED);
+	session->SetAnalyzerState(effective_tag, session::AnalyzerConfirmationState::CONFIRMED);
 
 	if ( ! analyzer_confirmation )
 		return;
 
-	const auto& tval = arg_tag ? arg_tag.AsVal() : tag.AsVal();
-	event_mgr.Enqueue(analyzer_confirmation, session->GetVal(), tval, val_mgr->Count(0));
+	event_mgr.Enqueue(analyzer_confirmation, session->GetVal(), effective_tag.AsVal(),
+	                  val_mgr->Count(0));
 	}
 
 void Analyzer::AnalyzerViolation(const char* reason, session::Session* session, const char* data,
                                  int len, zeek::Tag arg_tag)
 	{
+	auto const& effective_tag = arg_tag ? arg_tag : GetAnalyzerTag();
```

Comment:
```suggestion
 const auto& effective_tag = arg_tag ? arg_tag : GetAnalyzerTag();
```

Reply:
Again.. :-(

Seems with clang-format 14 there's now a QualifierAlignment option to help me ..